packages/modal-ui/src/lib/components/WalletNotInstalled.tsx

```diff
@@ -1,25 +1,29 @@
 import React from "react";
-import { Wallet } from "@near-wallet-selector/core";
+import { ModuleState } from "@near-wallet-selector/core";
 
 interface WalletNotInstalledProps {
-  wallet: Wallet;
+  module: ModuleState & {
+    metadata: {
+      downloadUrl?: string;
+    };
+  };
```

Comment:
We shouldn't need to do this, but it looks like I didn't quite get the types right with `ModuleState` 🙁  This is fine for now

Reply:
Yeah the `downloadUrl` is not being added to the metadata of `ModuleState` properly to keep typescript "happy" I added this kind of workaround.